CMakeLists.txt

```diff
@@ -99,8 +99,7 @@ set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 if(${IREE_BUILD_COMPILER})
   # Compiler dependencies require Python.
   # Find version 3+ first, so CMake doesn't find a lower version like 2.7 first.
-  find_package(PythonInterp 3 REQUIRED)
-  find_package(PythonLibs 3 REQUIRED)
+  find_package(Python3 REQUIRED)
```

Comment:
Did you mean to delete these? I don't really understand what these do exactly, but might be important :-P

Reply:
https://cmake.org/cmake/help/latest/module/FindPythonInterp.html is deprecated in favor of https://cmake.org/cmake/help/latest/module/FindPython3.html#module:FindPython3

This sort of change would be worth pulling into a separate commit or PR.

Reply:
Both PythonInterp and PythonLIbs are deprecated for our supported version of cmake (it won't detect newer versions of python3). Python3 is the currently support method.